.github/workflows/format.yml

```diff
@@ -0,0 +1,24 @@
+name: Format
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
```suggestion
```

I think we can just run this on every branch.

Reply:
Done